app/src/main/java/com/github/se/gomeet/ui/mainscreens/create/PublicCreate.kt

```diff
@@ -0,0 +1,175 @@
+import com.github.se.gomeet.ui.navigation.Route
+import com.github.se.gomeet.ui.navigation.TOP_LEVEL_DESTINATIONS
+import com.github.se.gomeet.viewmodel.EventViewModel
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeParseException
+
+@Composable
+fun PublicCreate(nav: NavigationActions, eventViewModel: EventViewModel) {
+
+  val titleState = remember { mutableStateOf("") }
+  val descriptionState = remember { mutableStateOf("") }
+  val locationState = remember { mutableStateOf("") }
+  val textDate = remember { mutableStateOf("") }
+  var dateState by remember { mutableStateOf<LocalDate?>(null) }
+  var dateFormatError by remember { mutableStateOf(false) }
+  val price by remember { mutableDoubleStateOf(0.0) }
+  val url = remember { mutableStateOf("") }
+
+  Scaffold(
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { selectedTab ->
+              nav.navigateTo(TOP_LEVEL_DESTINATIONS.first { it.route == selectedTab })
+            },
+            tabList = TOP_LEVEL_DESTINATIONS,
+            selectedItem = Route.EXPLORE)
+      }) { innerPadding ->
+        Text(
+            modifier = Modifier.padding(horizontal = 15.dp, vertical = 30.dp),
+            text = "Create",
+            style =
+                TextStyle(
+                    fontSize = 24.sp,
+                    lineHeight = 16.sp,
+                    fontFamily = FontFamily(Font(R.font.roboto)),
+                    fontWeight = FontWeight(1000),
+                    color = Color(0xFF073F57),
+                    textAlign = TextAlign.Center,
+                    letterSpacing = 0.5.sp,
+                ))
+        Column(
+            Modifier.padding(innerPadding),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center) {
+              Spacer(modifier = Modifier.height(100.dp))
+
+              OutlinedTextField(
+                  value = titleState.value,
+                  onValueChange = { newVal -> titleState.value = newVal },
+                  label = { Text("Title") },
+                  placeholder = { Text("Name the event") },
+                  modifier = Modifier.fillMaxWidth().padding(7.dp))
+
+              OutlinedTextField(
+                  value = descriptionState.value,
+                  onValueChange = { newVal -> descriptionState.value = newVal },
+                  label = { Text("Description") },
+                  placeholder = { Text("Describe the task") },
+                  modifier = Modifier.fillMaxWidth().padding(7.dp))
+
+              OutlinedTextField(
+                  value = locationState.value,
+                  onValueChange = { newVal -> locationState.value = newVal },
+                  label = { Text("Location") },
+                  placeholder = { Text("Enter an address") },
+                  modifier = Modifier.fillMaxWidth().padding(7.dp))
```

Comment:
A good idea would be suggesting locations based on the user input like in the bootcamp